.gitignore

```diff
@@ -18,3 +18,4 @@ origin.iml
 *.go~
 tags
 .envrc
+applyconfiguration-gen
```

Comment:
Just bear in mind that this also applies to any directory that happens to contain this name, so you're also ignoring https://github.com/kubernetes/code-generator/tree/master/cmd/applyconfiguration-gen within the vendor.

I'd recommend building the binary into an `_output` directory and ignoring that within `.gitignore`

This is an issue for models-schema on the API repo too which we should fix

Reply:
We build in the root in many (most?) operators.  I fixed this for /applygen

Reply:
I guess they don't have conflicting names? 🤷